sdk/healthdataaiservices/Azure.ResourceManager.HealthDataAIServices/api/Azure.ResourceManager.HealthDataAIServices.netstandard2.0.cs

```diff
@@ -280,7 +294,7 @@ public partial class HealthDataAIServicesPrivateLinkServiceConnectionState : Sys
     public partial class ManagedServiceIdentityUpdate : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentityUpdate>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentityUpdate>
     {
         public ManagedServiceIdentityUpdate() { }
-        public Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentityType? IdentityType { get { throw null; } set { } }
+        public Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentityType? Type { get { throw null; } set { } }
```

Comment:
rename to IdentityType

Reply:
we actually cannot do this because of typespec limitation.
I will introduce some customized code to replace the identity type completely so that this will no longer generate.